actionview/test/template/url_helper_test.rb

```diff
@@ -15,6 +15,10 @@ class UrlHelperTest < ActiveSupport::TestCase
     get "/other" => "foo#other"
     get "/article/:id" => "foo#article", :as => :article
     get "/category/:category" => "foo#category"
+
+    scope :engine do
+      get "/" => "foo#bar"
```

Comment:
@JanBussieck I tried this, this will not solve the problem with scope, test will still fail